samples/model-builder/create_batch_prediction_job_dedicated_resources_sample.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Sequence, Union
+
+from google.cloud import aiplatform, aiplatform_v1
+
+
+#  [START aiplatform_sdk_create_batch_prediction_job_dedicated_resources_sample]
+def create_batch_prediction_job_dedicated_resources_sample(
+    project: str,
+    location: str,
+    model_resource_name: str,
+    job_display_name: str,
+    gcs_source: Union[str, Sequence[str]],
+    gcs_destination: str,
+    machine_type: str = "n1-standard-2",
+    accelerator_count: int = 1,
+    accelerator_type: aiplatform_v1.AcceleratorType = aiplatform.gapic.AcceleratorType.NVIDIA_TESLA_K80,
```

Comment:
We've used string elsewhere:

https://github.com/googleapis/python-aiplatform/blob/master/samples/model-builder/create_training_pipeline_custom_container_job_sample.py#L33

 I think we can do the same here.

Reply:
Sg, I'll update to use string.